### DISCUSSION TITLE
Correct Process when adjusting values in the "constants" file

### DISCUSSION THREAD

**Discussion by kenika** [kennetek/gridfinity-rebuilt-openscad#84] [General]
When I wanted to update the magnet radius for press to fit, I first realized I couldn't do in the actual bin file. Then after looking a bit around in the code, I saw the separate "gridfinity constants" file which i could also use to adjust it directly in the openscad. 

But when i adjust the value in openscad it creates a custom value set which is then again not recognized when i create bins.

I cam around that by editing the "gridfinity constants" file with a text explorer and saving it. Is this the suggested way to go, or am I simply overthinking this and there is a much easier way also without deleting the default values for good?

**Comment by kennetek** [comment]
I don't really get how the custom value sets work since I do not use the customizer, but yes that is the intended way. Just edit the file directly. You don't necessarily need to use a text editor, you can use OpenSCAD, but since the constants file doesn't compile, it does not really matter how you edit it. If you want to keep the default values, you can always add a comment (using `//`) to comment out the old value and insert your new one. Additionally, this online repository will always have the default values, so there is no need to worry about losing them for good.
